src/Http/Controllers/SwaggerController.php

```diff
@@ -2,9 +2,10 @@
 
 namespace L5Swagger\Http\Controllers;
 
-use File;
-use Request;
-use Response;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Request;
+use Illuminate\Support\Facades\Response;
+use L5Swagger\Exceptions\L5SwaggerException;
 use L5Swagger\Generator;
```

Comment:
Move to the top. Namespaces should be ordered by length.

Reply:
OK, I'll do this ASAP